hibernate-reactive-core/src/main/java/org/hibernate/reactive/engine/impl/ReactiveEntityIdentityInsertAction.java

```diff
@@ -45,19 +44,20 @@ public void execute() throws HibernateException {
 
 	@Override
 	public CompletionStage<Void> reactiveExecute() throws HibernateException {
```

Comment:
The change to this method was necessary to ensure that identity ID values were obtained before completing `persist`.